packages/wonder-blocks-core/components/clickable.js

```diff
@@ -0,0 +1,141 @@
+// @flow
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
+
+import {Link} from "react-router-dom";
+import addStyle from "../util/add-style.js";
+import getClickableBehavior from "../util/get-clickable-behavior.js";
+import type {ClickableRole, State} from "./clickable-behavior.js";
+import type {StyleType} from "../util/types.js";
+
+type Props = {|
+    "aria-label": string,
+    children: (state: State) => React.Node,
+    disabled: boolean,
+    href?: string,
+    onClick?: (e: SyntheticEvent<>) => mixed,
+    role?: ClickableRole,
+    skipClientNav?: boolean,
+    style?: StyleType,
+    testId?: string,
+|};
+
+const StyledAnchor = addStyle<"a">("a");
+const StyledButton = addStyle<"button">("button");
+const StyledLink = addStyle<typeof Link>(Link);
+
+export default class Clickable extends React.Component<Props> {
```

Comment:
Can you add a detailed comment here explaining how this component should be used? I think you can move the comment that you currently have at the top of `clickable.md` into here and maybe expand it a bit explaining the usage and especially explaining when you should be using it and not Link or Button.